support-frontend/assets/helpers/redux/commonState/reducer.ts

```diff
@@ -88,40 +64,11 @@ export const commonSlice = createSlice({
 		setContributionTypes(state, action: PayloadAction<ContributionTypes>) {
 			state.settings.contributionTypes = action.payload;
 		},
-		setCurrencyId(state, action: PayloadAction<boolean>) {
+		setCurrencyId(state) {
 			state.internationalisation.currencyId = getLocalisedCurrencyId(
 				state.internationalisation,
```

Comment:
Given that `getLocalisedCurrencyId` is now so simple, I'd suggest removing it altogether
```suggestion
		setCurrencyId(state) {
			state.internationalisation.currencyId = state.internationalisation.defaultCurrency
```
It's also worth thinking about whether we need this reducer at all anymore, given that it's just setting information in one bit of the state to another bit of the state.

Reply:
Many thanks Imogen! I've checked the usage of `internationalisation.currencyId` - it appears 13 times across 8 files, which include files involved in paper subscription checkout, subscription forms for Guardian Weekly, etc. On reflection, I don't want to touch those files now but instead just implement your suggested change above and leave it at that.